### ISSUE TITLE
Bot comments

### ISSUE THREAD

**Issue by dmitry-engineer** [peter-evans/find-comment#52]
I try to find comments by github-action bot but can't Receive empty outputs

**Comment by dmitry-engineer** [comment]
```
- name: Find Comment
  uses: peter-evans/find-comment@v1
  continue-on-error: true
  id: fc
  with:
    issue-number: ${{ github.event.pull_request.number }}
    comment-author: 'github-actions[bot]'
    direction: last
- run: |
    echo "${{ steps.fc.outputs.comment-id }}"
```

**Comment by dmitry-engineer** [comment]
The problem is that i try to find commit comment but not PR comment

**Comment by dmitry-engineer** [comment]
I use your GitHub action `comment-commit`. Is it possible to find comment to a commit?

**Comment by peter-evans** [comment]
Hi @dmitry-engineer 

> Is it possible to find comment to a commit?

Unfortunately it's not possible with this action. Commit comments are treated differently to issue and pull request commits in GitHub's API. This action only supports issue and pull request comments.

I tried to search for other actions on the GitHub marketplace that might help you.
The only one I found was this one: https://github.com/dlouisenz/last-commit-message-text-finder-action
